### PR TITLE
Run docker builds as the host user

### PR DESCRIPTION
**Pull request by ghost**
DO NOT MERGE

### CODE REVIEW

**Comment by rjones-lf** [comment]
reverify jenkins

File: Makefile [hunk 1]
```diff
@@ -113,6 +114,7 @@ $(GOPATH)/bin/%:
 	@echo "Building $@"
 	@mkdir -p $(@D)
 	@docker run -i \
+		--user=$(UID)
 		-v $(abspath vendor/github.com/golang/protobuf):/opt/gopath/src/github.com/golang/protobuf \
```
Comment (ghaskins) [inline Makefile:118]: note that you need a continuation "\" at the end of the line in both places, that is why the "docker run" thing is complaining

Comment (ghost) [inline Makefile:118]: thanks

**Comment by rjones-lf** [comment]
reverify jenkins

**Comment by rjones-lf** [comment]
reverify jenkins

**Comment by rjones-lf** [comment]
reverify jenkins

**Pull request closed without merging**